Dockerfile

```diff
@@ -1 +1,7 @@
-FROM golang:1.4.1-onbuild
+FROM golang:1.6-alpine
+
+MAINTAINER Tom Denham <tom@tigera.io>
+
+RUN apk -U add bash git musl-dev gcc linux-headers
+WORKDIR /go/src/github.com/coreos/flannel
+
```

Comment:
That sounds cool, but I don't think it's a requirement for getting this PR merged. I'm happy to change golang:1.6-alpine to something else if you think that would make future cross compilation easier. However, it looks like alpine is probably fine - e.g. see https://github.com/tianon/rawdns/blob/master/Dockerfile.cross